gf-webapp-admin/src/component/rooms/room/deleteRoom/DeleteRoom.tsx

```diff
@@ -0,0 +1,24 @@
+import {Button} from "gf-ui-lib/components/Button/Button";
+import {useNavigate} from "react-router-dom";
+import {deleteRoom} from "src/service/RoomService";
+
+interface DeleteRoomProps {
+  id: string;
+}
+
+export const DeleteRoom = (props: DeleteRoomProps) => {
+  const navigate = useNavigate();
+
+  const onDeleteRoom = async () => {
+    await deleteRoom(props.id);
+    navigate("/rooms");
```

Comment:
hardcode

Reply:
remade `/rooms` to `MAIN_PAGE_ROUTE` (imported)